solutions/silver/cf-702C.mdx

```diff
@@ -137,7 +137,50 @@ public class CellularNetwork {
 ```
 
 </JavaSection>
+<PySection>
+
+```py
+"""
+returns the first index in the array that is >= value, or len(arr)
+if no such index exists
+"""
```

Comment:
again, doc comments go under the function

Reply:
my bad

Reply:
nah ur good :heart: